mne_bids/tests/test_mne_bids.py

```diff
@@ -182,21 +189,47 @@ def test_kit():
     # ensure the channels file has no STI 014 channel:
     channels_tsv = make_bids_basename(
         subject=subject_id, session=session_id, task=task, run=run,
-        suffix='channels.tsv', acquisition=acq,
+        suffix='channels.tsv',
```

Comment:
what happens if someone provides an `acq`? will an error be raised?

Reply:
yeah, I am not entirely sure how to handle that...